export_img.py

```diff
@@ -1,13 +1,10 @@
 """
-Program export images in svg format to png format.
+Program export images in SVG format to PNG format.
```

Comment:
Druhy radek popisku neodpovida pozmenenemu pouziti.
Myslim si take, ze tento skript vyzaduje nainstalovany Inkscape na pocitaci - to je vhodne zminit.

Reply:
nechápu větu - neodpovídá pozměněnému použití. To jsem změnila na veliké písmena aby to bylo přehledné, co je špatně?

Reply:
a mělo by být exports že?

Reply:
jo překlep opravím, ale co nedává smysl nevím, možná jen přeházet slova jinak :-D

Reply:
@JureckovaK  díky, já koukala zběžně, už chápu :)